README.md

```diff
@@ -381,3 +381,15 @@ selectable:
         }
     }
 ```
+
+## Example: Allow specific NodeTypes in the Placeholder-Insert
+
+By default, all Form-Element NodeTypes are visible in the Placeholder-Insert (except StaticText and Section).
+
+If you want to hide specific Form-Elements from the Placeholder-Insert then just disable the superType.
+
+```yaml
+'FooBar.Form.Builder:Element.HiddenInInset':
```

Comment:
```suggestion
'Some.Package.Form.Builder:Element.HiddenInInset':
```

I would use `Some.Package` here. As we use this also at other places in the documentation. Just to keep it consistent :)

Reply:
And there's a typo: `Inset` instead of `Insert`